lighthouse-core/lib/proto-preprocessor.js

```diff
@@ -22,6 +22,16 @@ function processForProto(result) {
   /** @type {LH.Result} */
   const reportJson = JSON.parse(result);
 
+  // Clean up the configSettings
+  if ('configSettings' in reportJson) {
```

Comment:
don't need to guard this
```suggestion
  if (reportJson.configSettings) {
```